src/votes/services.py

```diff
@@ -33,26 +31,30 @@ def get_votes_by_parliament_session(parliament_session):
             vote_details_soup = soupUtils.getSoupFromLink(
                 details_link + str(vote_record['althingi_id'])
             )
-            parliament_member_votes = get_parliament_member_votes(vote_details_soup)
+            parliament_member_votes = get_parliament_member_votes(
+                vote_details_soup
+            )
             for parliament_member_vote in parliament_member_votes:
                 parliament_member_name = get_parliament_member_name_from_vote(
                     parliament_member_vote
                 )
                 parliament_member = ParliamentMember.objects.filter(
                     name=parliament_member_name
                 )
-                if parliament_member.exists():
-                    vote_result = get_parliament_member_result_from_vote(parliament_member_vote)
-                    Vote.objects.create(
-                        parliament_member=parliament_member.get(),
-                        althingi_result=vote_result,
-                        vote_record=created_vote_record
+                # Create parliamentMember if he does not exist
+                if not parliament_member.exists():
```

Comment:
Are there any more services that we need to account for the "create parliament member if not exists"?